File: src/main/java/br/com/rcc_dev/testes/borders/SoapBorder.java
package br.com.rcc_dev.testes.borders;

import io.vertx.core.Vertx;
import io.vertx.ext.web.Router;

public class SoapBorder {
  
  // TODO: criar as bordas SOAP
  public Router createRouter(Vertx vertx){
    Router router = Router.router(vertx);
    
    return router;
  }
  
}

File: src/main/kotlin/br/com/rcc_dev/testes/borders/SoapBorder.kt
package br.com.rcc_dev.testes.borders

import io.vertx.core.Vertx
import io.vertx.ext.web.Router

class SoapBorder {

  fun createRouter(vertx: Vertx): Router {
    val router = Router.router(vertx)

    return router
  }

}